.github/workflows/build_binaries.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and publish binaries
+on:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    name: Build and publish binaries
+    strategy:
+      matrix:
+        go-version: [ 1.16.x ]
+        platform: [ ubuntu-latest ]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
```

Comment:
there is a v2 or v3 now: https://github.com/marketplace/actions/setup-go-environment

Reply:
Ok, updated to v2.